### PR TITLE
Add GBP currency

### PR DESCRIPTION
**Pull request by jezen**

### CODE REVIEW

File: src/Data/Currency/Currencies.hs [hunk 1]
```diff
@@ -65,8 +66,11 @@ data CLP = CLP deriving (Show, Read, Eq)
 
 -- | European Union Euro
 data EUR = EUR deriving (Show, Read, Eq)
+--
+-- | British Poung Sterling
```
Comment (alx741) [inline src/Data/Currency/Currencies.hs]: Poung -> Pound

**Review by alx741 — CHANGES_REQUESTED** [review]

**Comment by alx741** [comment]
Hey thanks a lot for this one!, could you get that typo in pound? so I can merge it then :)

File: src/Data/Currency/Currencies.hs (hunk 1 above)
Comment (jezen) [reply src/Data/Currency/Currencies.hs]: Oh my goodness. I am useless without a compiler. Yes, I'll fix that. Thanks for catching it.

**Review by alx741 — APPROVED** [review]

**Comment by alx741** [comment]
Perfect, thank you!

**Pull request merged**